aeron-tools/src/main/java/uk/co/real_logic/aeron/tools/perf_tools/AeronPing.java

```diff
@@ -0,0 +1,378 @@
+/*
```

Comment:
What does this and Pong do differently than the samples Ping and Pong?

Also, why does this do its own calculations instead of using HdrHistogram?